tests/all-test.php

```diff
@@ -103,30 +101,113 @@ public function setUp() {
 			'track_page_types'          => array( 'page' ),
 		);
 		update_option( 'parsely', $option_defaults );
-		self::$parsely_html = <<<PARSELYJS
-<script data-cfasync="false" id="parsely-cfg" data-parsely-site="blog.parsely.com" src="https://cdn.parsely.com/keys/blog.parsely.com/p.js"></script>
-PARSELYJS;
 	}
 
+	public function test_class_version() {
+		self::assertEquals( '2.4.1', \Parsely::VERSION );
+	}
+
+	public function test_constant_version() {
+		self::assertEquals( \Parsely::VERSION, PARSELY_VERSION );
+	}
+
+	public function test_cache_buster() {
+		self::assertEquals( PARSELY_VERSION, \Parsely::get_asset_cache_buster() );
+	}
+
+	public function test_plugin_url_constant() {
+		self::assertTrue( defined( 'PARSELY_PLUGIN_URL' ) && is_string( PARSELY_PLUGIN_URL ) && strlen( PARSELY_PLUGIN_URL ) > 0 );
+	}
 
 	/**
-	 * Test the parsely tag.
-	 *
-	 * @category   Function
-	 * @package    SampleTest
+	 * Test the default parsely script enqueues.
 	 */
-	public function test_parsely_tag() {
-		echo 'NOW IT BEGINS';
+	public function test_parsely_default_scripts() {
 		ob_start();
 		$post_array = $this->create_test_post_array();
 		$post       = $this->factory->post->create( $post_array );
 		$this->go_to( '/?p=' . $post );
-		echo esc_html( self::$parsely->insert_parsely_javascript() );
+		echo self::$parsely->insert_parsely_javascript();
+
+		$intermediate_output = ob_get_contents();
+		self::assertEquals(
```

Comment:
```suggestion
		self::assertSame(
```

Does a type check (`===`) as well, instead of just the value check (`==`).